src/main/java/reactor/netty/http/client/HttpClient.java

```diff
@@ -830,6 +830,20 @@ public final HttpClient wiretap(boolean enable) {
 	 * @return a new {@link HttpClient}
 	 */
 	public final HttpClient metrics(boolean metricsEnabled) {
+		return metrics(metricsEnabled, reactor.netty.Metrics.MAX_URI_TAGS);
+	}
```

Comment:
Now that there is an overloaded method, the Javadoc for this one should probably mention the default value used for maxUriTags.